bigquery/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # When running this Dockerfile context must be project root, in order to pick
 # up indexer_util and dataset_config
 
-FROM python:2
+FROM python:3
```

Comment:
Please also test that this works on GCP, in your test project(s).

- 1000 Genomes
- Framingham
- The new AMP PD data
- NHS
- baseline_second_data_release

If you could leave the clusters around so I can double-check, that would be great.